packages/ui5-tooling-modules/README.md

```diff
@@ -46,6 +46,9 @@ npm install ui5-tooling-modules --save-dev
 - *removeScopePreceder*: `boolean`
   Removes the scope preceder `@` from the namespace/path of the 3rd party module when adding it to the namespace with the *addToNamespace* option.
```

Comment:
What is a "preceder"? A "prefix", maybe? :-)

Reply:
Looking into the [documentation of NPM](https://docs.npmjs.com/cli/v9/using-npm/scope) it is stated like that:

*When used in package names, scopes are preceded by an @ symbol and followed by a slash, e.g.*

*preceded* => **preceder**

But somehow my research wasn't careful enough and it noun **preceder** doesn't exist in English - so'll adopt back to prefix...